getting_started/cloud_infrastructure_access/dedicated-aws-access.adoc

```diff
@@ -0,0 +1,56 @@
+[id="dedicated-aws-access"]
+= AWS Infrastructure Access
+include::modules/common-attributes.adoc[]
+:context: dedicated-aws-access
+
+AWS infrastructure access allows link:https://access.redhat.com/node/3610411[Customer Portal Organization Admins] and cluster owners to enable Amazon Web Services (AWS) Identity and Access Management (IAM) users to have federated access to the AWS Console for their {product-title} cluster. Admins may select between Network Management or Read-only access options.
+
+[id="dedicated-configuring-aws-access"]
+== Configuring AWS Infrastructure Access
+
+.Prerequisites
```

Comment:
Prereqs should be unordered bullets, rather than numbers.
Source: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#writing-tasks